CODEOWNERS

```diff
@@ -43,3 +43,7 @@ src/sfctl/custom_chaos_schedule.py @likevi
 
 # Container related
 src/sfctl/custom_container.py @leikong
+
+# Upgrade related
```

Comment:
nit: can you name this Cluster Upgrade related please? Thanks!

Reply:
Sure